<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>371</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Positon layout</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>355</width>
       <height>244</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>No of sides</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Marking patterns</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Dummy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>4 corners</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8 corners</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fully automatic</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_3">
        <item>
         <property name="text">
          <string>Follow boundary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fastest way</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>No of dummy  positions</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Chip orientation</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Micromanager Version</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="mm_version_layout">
        <item>
         <widget class="QRadioButton" name="mm20_button">
          <property name="text">
           <string>2.0</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mm14_button">
          <property name="text">
           <string>1.4</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="chip_orientation_layout">
        <item>
         <widget class="QRadioButton" name="chip_vertical_button">
          <property name="text">
           <string>vertical</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="chip_horizontal_button">
          <property name="text">
           <string>horizontal</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="grid_type_layout">
        <item>
         <widget class="QRadioButton" name="one_rect_button">
          <property name="text">
           <string>One Side</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="two_rect_button">
          <property name="text">
           <string>Two sides</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Number of rows</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Number of columns</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Mark positions</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>232</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="one_side_layout">
      <item row="3" column="0">
       <widget class="QPushButton" name="bl_button">
        <property name="text">
         <string>Bottom Left 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="tl_button">
        <property name="text">
         <string>Top Left 1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="tr_button">
        <property name="text">
         <string>Top Right 1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="br_button">
        <property name="text">
         <string>Bottom Right 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="tl_button_2">
        <property name="text">
         <string>Top Left 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="tr_button_2">
        <property name="text">
         <string>Top Right 2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="bl_button_2">
        <property name="text">
         <string>Bottom Left 2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="br_button_2">
        <property name="text">
         <string>Bottom Right 2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Rules</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
